vms/platformvm/service_test.go

```diff
@@ -44,6 +44,7 @@ import (
 
 	vmkeystore "github.com/ava-labs/avalanchego/vms/components/keystore"
 	pchainapi "github.com/ava-labs/avalanchego/vms/platformvm/api"
+	"github.com/ava-labs/avalanchego/vms/platformvm/block/builder"
```

Comment:
Can e move this up with the other unnamed imports?

Reply:
`gci` can't come soon enough